added package chemical elements (#44)
espanso/hub: packages/chemical-elements/0.1.0/README.md

> A package containing all currently known chemical elements and its abreviations.

Reviewer: ```suggestion
A package containing all currently known chemical elements and their abreviations.
```

> Just type the chemical symbol preceded by semicolon and it will be replaced by its name.

Reviewer: ```suggestion
Just type the chemical symbol preceded by a colon and it will be replaced by its name.
```

Or, should alternatively all the triggers be updated to use `;`?

Author: colon is the correct word

> Example: ":O" will be replaced by "Oxigen"

Reviewer: ```suggestion
Example: ":O" will be replaced by "Oxygen"
```
